MrBacon8742/school-website-project
# feat: Upload entire website to main

Fixes #1 

## Summary of Changes
Completely uploaded website to main branch.

## Additional context
There is additional progress being done. Major changes will get a pull request later.

## Testing instructions
Download all files and run index.html. Ensure all features in the code work as expected.
